rocAL/rocAL_pybind/example/new_api/READERS_TEST_FILE.sh

```diff
@@ -0,0 +1,251 @@
+    python$ver rocAL_api_python_unittest.py \
+        --image-dataset-path $data_dir \
+        --augmentation-name snow \
+        --batch-size $batch_size \
+        --display \
+        --NHWC \
+        --local-rank 0 \
+        --world-size $gpus_per_node \
+        --num-threads 1 \
+        --num-epochs 2 2>&1 | tee -a run.log.rocAL_api_python_unittest.${CURRENTDATE}
+fi
+####################################################################################################################################
+
+
+####################################################################################################################################
+if [[ rocAL_api_coco_pipeline -eq 1 ]]; then
+
+    # Mention dataset_path
+    data_dir=$ROCAL_DATA_PATH/coco/coco_10_img/val_10images_2017/
+
+
+    # Mention json path
+    json_path=$ROCAL_DATA_PATH/coco/coco_10_img/annotations/instances_val2017.json
+
+    # rocAL_api_coco_pipeline.py
+    # By default : cpu backend, NCHW format , fp32
+    # Please pass annotation path in addition to other common args
+    # Annotation must be a json file
+    python$ver rocAL_api_coco_pipeline.py \
+        --image-dataset-path $data_dir \
+        --json-path $json_path \
+        --batch-size $batch_size \
+        --display \
+        --rocal-gpu \
+        --NHWC \
+        --local-rank 0 \
+        --world-size $gpus_per_node \
+        --num-threads 1 \
+        --num-epochs 1 2>&1 | tee -a run.log.rocAL_api_coco_pipeline.${CURRENTDATE}.txt
+fi
+####################################################################################################################################
+
+
+####################################################################################################################################
+if [[ rocAL_api_caffe_reader -eq 1 ]]; then
+
+    # Mention dataset_path
+    # Classification
+    data_dir=$ROCAL_DATA_PATH/caffe/caffe_classification/ilsvrc12_train_lmdb/
+
+    # rocAL_api_caffe_reader.py
+    # By default : cpu backend, NCHW format , fp32
+    # use --classification for Classification / --no-classification for Detection
+
+    python$ver rocAL_api_caffe_reader.py \
+        --image-dataset-path $data_dir \
+        --classification \
+        --batch-size $batch_size \
+        --display \
+        --rocal-gpu \
+        --NHWC \
+        --local-rank 0 \
+        --world-size $gpus_per_node \
+        --num-threads 1 \
+        --num-epochs 1 2>&1 | tee -a run.log.rocAL_api_caffe_reader_classification.${CURRENTDATE}.txt
+fi
+####################################################################################################################################
+
+
+
+####################################################################################################################################
+if [[ rocAL_api_caffe_reader -eq 1 ]]; then
+
+    # Mention dataset_path
+    # Detection
+    data_dir=$ROCAL_DATA_PATH/caffe/caffe_detection/lmdb_record/
```

Comment:
@swetha097 Same missing "/rocal_data/"

Reply:
we need to export ROCAL_DATA_PATH=/Absolute/Path/Of/rocal_data/

We can see the dumped image outputs in OUTPUTS_IMAGES_PYTHON/NEW_API/